parameter_set:
  bframe: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  uframe: [1.0, 0.0, 0.7, 0.0, 0.0, 0.0]
  tframe: [0.13, 0.0, 0.075, 3.141592653589793, 2.094395102393195, 3.141592653589793]
  endstroke_neg: [-3.141592653589793, -1.047197551196598, -2.967059728390360, -47.123889803846900, -2.094395102393195, -47.123889803846900]
  endstroke_pos: [3.141592653589793, 2.705260340591211, 1.919862177193763, 47.123889803846900, 2.094395102393195, 47.123889803846900]
  q_avg: [0.0, 0.829031394697307, -1.570796326794897, 0.0, -0.523598775598299, 0.0]
  max_joint_speed: [2.70526034, 2.70526034, 2.967059728, 6.283185307, 6.10865238, 9.599310886]
  urdf_pkg_name: 'ns16hand_description'
  urdf_file_name: 'NS16_base_with-flange-ref.URDF'
  trj_sampling_time: 0.1
  default_initial_position: [1.0, 0.0, 0.8, 0.0, 1.57, 0.0]
  reorient_gain: 1.0